packages/shared-components/src/components/onto-language-selector/onto-language-selector.tsx

```diff
@@ -14,6 +14,12 @@ export class OntoLanguageSelector {
   private items: DropdownItem<string>[] = [];
   private readonly onLanguageChangeSubscription: () => void;
 
+  /**
+   * Specifies the dropdown items' alignment. If not provided, the items and the dropdown button will be aligned to the left.
+   *
+   */
+  @Prop() dropdownAlignment: DropdownItemAlignment = DropdownItemAlignment.LEFT;
```

Comment:
This has to be set to `DropdownItemAlignment.RIGHT` or configured within the header component. Currently, a horizontal scroll appears when the language selector is opened.
![image](https://github.com/user-attachments/assets/f24cf832-e05d-479d-b9b9-491cc20de4e1)

Reply:
fixed